packages/amplication-client/src/Plugins/InstalledPlugins.tsx

```diff
@@ -56,31 +56,35 @@ const InstalledPlugins: React.FC<Props> = ({ match }: Props) => {
   );
 
   const onOrderChange = useCallback(
```

Comment:
because of growing in plugin logic, I think all the logic should be part of the plugin hook